README.md

```diff
@@ -1 +1,21 @@
-# cx-ressources
\ No newline at end of file
+# Catena-X Resources
+
+This repository offers various Catena-X contents to download.
+
+## Use of Catena-X Labels
+
+To use the labels, you **must**:
+
+1. Downloaded the contract [HERE](./static/Logo%20Use%20Agreement_V2_BOtOG.pdf)
```

Comment:
I would change "contract" to "Logo Use Agreement" - it makes it more specific and matches the document.

Reply:
Changed it everywhere :)